packages/harpguru-core/src/components/option/option.tsx

```diff
@@ -1,21 +1,16 @@
 import { useTimingTransition } from 'react-native-redash'
 import Animated, { Easing, interpolate } from 'react-native-reanimated'
-import {
-  PanGestureHandler,
-  TapGestureHandler,
-  State,
-} from 'react-native-gesture-handler'
-import type {
-  PanGestureHandlerGestureEvent,
-  TapGestureHandlerStateChangeEvent,
-} from 'react-native-gesture-handler'
-import { View, Text, TextStyle } from 'react-native'
+import { PanGestureHandler, State } from 'react-native-gesture-handler'
+import type { PanGestureHandlerGestureEvent } from 'react-native-gesture-handler'
+import { View, Text } from 'react-native'
 import React, { useState } from 'react'
 
+import { OptionValue } from '../option-value'
 import { OptionIds } from '../../types'
 import { getSizes } from '../../styles'
 import { usePrevious } from '../../hooks'
 
+import { setOptionsInListOfSix } from './utils'
```

Comment:
This function deserves a better name.

Reply:
I think it's ok actually. Especially now that it's internal naming is much better.